CITATION.cff

```diff
@@ -0,0 +1,27 @@
+# This CITATION.cff file was generated with cffinit.
+# Please do not update this file manually. Use https://bit.ly/cffinit instead!
+
+cff-version: 1.2.0
+title: metriq-gym
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - name: Unitary Foundation
+    address: 315 MONTGOMERY ST
+    city: San Francisco
+    post-code: '94104'
+    region: Califronia
+    country: US
+    location: 10TH FLOOR
```

Comment:
Do we really want the full address here?
nit: California is misspelled

Reply:
Probably don't need the full address, you're right. Also, good catch on the spelling mistake (although, I personally think that name sounds better). Fixed in [456dc61](https://github.com/unitaryfund/metriq-gym/pull/286/commits/456dc61c9b24ca13e2cb78281ad86df692909722).